Library/Homebrew/download_strategy.rb

```diff
@@ -95,10 +95,10 @@ def quiet?
   # @api public
   def stage(&block)
     UnpackStrategy.detect(cached_location,
-                          prioritise_extension: true,
+                          prioritize_extension: true,
```

Comment:
As mentioned elsewhere: let's not rename this or, if we want to discuss renaming, make a PR that only renames and does nothing else.

Reply:
https://github.com/Homebrew/brew/pull/14735#discussion_r1116446511